src/bugreporter/odemis_bugreporter.py

```diff
@@ -281,15 +281,20 @@ def compress_files(self):
                     # subprocess doesn't have timeout argument in python 2.x, so use future instead
                     f = self._executor.submit(subprocess.check_output, ['odemis-cli', '--list-prop', '*'])
                     props = f.result(60)
-                    hwfn = "/tmp/odemis-hw-status.txt"
-                    with open(hwfn, 'w+') as f:
-                        f.write(props)
-                    files.append(hwfn)
                 except Exception as ex:
                     logging.warning("Cannot save hw status: %s", ex)
 
+            # Save processes, kernel name, ip address
+            ps = subprocess.check_output(['ps', 'aux'])
```

Comment:
With `check_output()`, any failure to run the command will raise an exception, which will cause this whole function to abort. So I'd suggest to either protect each call with a try/except, or use a different subprocess function.

Reply:
done